### PR TITLE
Hotfix - Support `0` value

### PR DESCRIPTION
**Pull request by mrdezzods**
Currently, when the value is 0, it's completely ignored and not formatted. This change will format the 0 number values. whitespaces and empty stings are still ignored.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "alpinejs-money",
-  "version": "1.1.3",
+  "version": "1.2.0",
```
Comment (markmead) [inline package.json]: I wouldn't say this is a minor release, more of a patch.

**Review by markmead — CHANGES_REQUESTED** [review]
Can you add a PR description and an example, please.

File: package.json (hunk 1 above)
Comment (mrdezzods) [reply package.json]: Since it changes current behaviour, it can also be seen as minor

Comment (markmead) [reply package.json]: I disagree. It's more of a patch to the existing code than a change people should be aware of.

**Review by markmead — APPROVED** [review]

**Pull request merged**